models/qualtrics__directory.sql

```diff
@@ -68,10 +68,10 @@ agg_distributions as (
         count(distinct distribution.survey_id) as count_surveys_sent_30d
         
     from distribution_contact
-    join directory_contact 
+    join directory_contact
         on distribution_contact.contact_id = directory_contact.contact_id
         and distribution_contact.source_relation = directory_contact.source_relation
-    join distribution 
+    join distribution
```

Comment:
Same comment about explicitly defining the type of join.

Reply:
updated to left